src/OrbitCaptureClient/CaptureClient.cpp

```diff
@@ -66,7 +66,8 @@ Future<ErrorMessageOr<CaptureListener::CaptureOutcome>> CaptureClient::Capture(
     absl::flat_hash_map<uint64_t, FunctionInfo> selected_functions,
     TracepointInfoSet selected_tracepoints, absl::flat_hash_set<uint64_t> frame_track_function_ids,
     bool collect_thread_state, bool enable_introspection,
-    uint64_t max_local_marker_depth_per_command_buffer) {
+    uint64_t max_local_marker_depth_per_command_buffer, bool collect_memory_info,
```

Comment:
This function has a lot of arguments, it should be some struct, could you add a TODO about that?

Reply:
Done. Thanks for the suggestion!